src/measure_theory/ae_eq_fun.lean

```diff
@@ -276,10 +273,10 @@ comp₂_to_fun _ _ _ _
 instance : add_monoid (α →ₘ γ) :=
 { zero      := 0,
   add       := (+),
-  add_zero  := by rintros ⟨a⟩; exact quotient.sound (all_ae_of_all $ assume a, add_zero _),
-  zero_add  := by rintros ⟨a⟩; exact quotient.sound (all_ae_of_all $ assume a, zero_add _),
+  add_zero  := by rintros ⟨a⟩; exact quotient.sound (a_e_of_all _ $ assume a, add_zero _),
```

Comment:
I really prefer `ae` to `a_e` (it's written `a.e.` in papers, without space, so it's almost like it's a single word).

Reply:
Done.